spec/td/command/schema_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+require 'td/command/common'
+require 'td/config'
+require 'td/command/list'
+require 'td/command/schema'
+require 'td/client/model'
+require 'time'
+
+describe TreasureData::Command do
+  let(:client){ double('client') }
+  let(:table){ double('table', schema: schema) }
+  let(:db_name){ "database" }
+  let(:table_name){ "table" }
+  let(:command) { Class.new { include TreasureData::Command }.new }
+  let(:columns){ ["foo:int", "BAR\u3070\u30FC:string@bar", "baz:baz!:array<double>@baz"] }
```

Comment:
Looks their sql_alias aren't parsed correctly with this regex: https://github.com/treasure-data/td-client-ruby/pull/87/files#diff-3b5a4fa20c69ec6f36fb26b0df57906dR308

```ruby
puts RUBY_DESCRIPTION

["foo:int", "BAR\u3070\u30FC:string@bar", "baz:baz!:array<double>@baz"].each do |column|
  puts column
  unless /\A(?<name>.*)(?::(?<type>[^:]+))(?:@(?<sql_alias>[^:@]+))?\z/ =~ column
    raise "fail"
  end

  p $~
  puts
end
```

```
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
foo:int
#<MatchData "foo:int" name:"foo" type:"int" sql_alias:nil>

BARばー:string@bar
#<MatchData "BARばー:string@bar" name:"BARばー" type:"string@bar" sql_alias:nil>

baz:baz!:array<double>@baz
#<MatchData "baz:baz!:array<double>@baz" name:"baz:baz!" type:"array<double>@baz" sql_alias:nil>
```